### PR TITLE
feat: support team service accounts

### PR DESCRIPTION
**Pull request by junminahn**
- add team service account workflow

### CODE REVIEW

File: app/page-partials/my-dashboard/TeamInfoTabs.tsx [hunk 1]
```diff
@@ -477,6 +519,42 @@ function TeamInfoTabs({ alert, currentUser, team, loadTeams }: Props) {
...
+                      downloadText(prettyJSON(text), `${serviceAccount.clientId}.json`);
+                    }}
+                  >
+                    Download
+                  </Button>
+                )
+              ) : (
+                <Button
+                  onClick={async () => {
+                    const [sa, err] = await requestServiceAccount(team.id);
```
Comment (thegentlemanphysicist) [inline app/page-partials/my-dashboard/TeamInfoTabs.tsx:548]: Is this error handled anywhere?

File: lambda/app/src/routes.ts [hunk 2]
```diff
@@ -487,4 +490,34 @@ export const setRoutes = (app: any) => {
       handleError(res, err);
     }
   });
+
+  app.post(`${BASE_PATH}/teams/:id/service-accounts`, async (req, res) => {
```
Comment (thegentlemanphysicist) [inline lambda/app/src/routes.ts:494]: Maybe a naive question, but are we not able to modify or delete service accounts once created?

**Review by thegentlemanphysicist — COMMENTED** [review]
2 questions

File: lambda/app/src/routes.ts (hunk 2 above)
Comment (junminahn) [reply lambda/app/src/routes.ts:494]: they might update data in the future, but for now we don't have the business requirements.

Comment (thegentlemanphysicist) [reply lambda/app/src/routes.ts:494]: sounds good

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**